.github/workflows/mark-issue-stale.yml

```diff
@@ -20,3 +20,5 @@ jobs:
           days-before-close: -1
           # Get issues in ascending (oldest first) order.
           ascending: true
+          # Label to apply when issue is marked stale.
+          stale-issue-label: '[Status] - Stale'
```

Comment:
The existing label is `[Status] Stale` (no "-"). Let's use that one.

Reply:
Doh, silly me. Updated in a subsequent commit.